### PR TITLE
Partial txn redesign

### PR DESCRIPTION
**Pull request by laics1984**
- Partial transaction datatable
- cosign transaction and announce

### CODE REVIEW

File: src/modules/transaction/components/UnconfirmedTransactionDataTable.vue [hunk 1]
```diff
@@ -274,8 +274,7 @@ export default defineComponent({
       txnQueryParams.updateFieldOrder(blockDescOrderSortingField);
 
       let transactionSearchResult = await dashboardService.searchTxns(transactionGroupType.UNCONFIRMED, txnQueryParams);
-
-      let formattedTxns = await dashboardService.formatUnconfirmedMixedTxns(transactionSearchResult.transactions);
+      let formattedTxns = await dashboardService.formatUnconfirmedMixedTxns(transactionSearchResult.transactions);``
```
Comment (sleepyOwl14) [inline src/modules/transaction/components/UnconfirmedTransactionDataTable.vue:277]: ??

File: src/modules/transaction/components/PartialDashboardDataTable.vue [hunk 2]
```diff
@@ -159,14 +158,58 @@ export default{
...
+
+    let transactionGroupType = Helper.getTransactionGroupType();
+
+    let dashboardService = new DashboardService(walletState.currentLoggedInWallet, currentAccount);
+
+    let loadUnconfirmedTransactions = async() => {
+      let txnQueryParams = Helper.createTransactionQueryParams();
+      txnQueryParams.pageSize = 1;
+      txnQueryParams.address = currentAccount.address;
+      txnQueryParams.embedded = true;
```
Comment (sleepyOwl14) [inline src/modules/transaction/components/PartialDashboardDataTable.vue:182]: embedded should be false, will remove this line later

File: src/modules/transaction/components/PartialDashboardDataTable.vue [hunk 3]
```diff
@@ -159,14 +158,58 @@ export default{
...
+
+    let currentAccount = walletState.currentLoggedInWallet.selectDefaultAccount() ? walletState.currentLoggedInWallet.selectDefaultAccount() : walletState.currentLoggedInWallet.accounts[0];
+    currentAddress.value = currentAccount.address;
+    let blockDescOrderSortingField = Helper.createTransactionFieldOrder(Helper.getQueryParamOrder_v2().DESC, Helper.getTransactionSortField().BLOCK);
+
+    let transactionGroupType = Helper.getTransactionGroupType();
+
+    let dashboardService = new DashboardService(walletState.currentLoggedInWallet, currentAccount);
+
+    let loadUnconfirmedTransactions = async() => {
```
Comment (sleepyOwl14) [inline src/modules/transaction/components/PartialDashboardDataTable.vue:178]: will change it to `loadPartialTransactions` later, need to use the correct name

File: src/modules/transaction/components/UnconfirmedTransactionDataTable.vue (hunk 1 above)
Comment (laics1984) [reply src/modules/transaction/components/UnconfirmedTransactionDataTable.vue:277]: I think there is 2 extra ` character. likely added accidently

File: src/util/typeHelper.ts [hunk 4]
```diff
@@ -186,6 +187,10 @@ export class Helper{
             hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false,
           }).format(date);
       }
+
+      static formatDeadline(timestamp: number) :string{
+        return new Date(timestamp + Deadline.timestampNemesisBlock * 1000).toLocaleString()
```
Comment (sleepyOwl14) [inline src/util/typeHelper.ts:192]: why use toLocaleString ? it will have language specify characters. nvm, I will fix it later

**Review by sleepyOwl14 — APPROVED** [review]
the needed changes will be update later

**Pull request merged**